### PR TITLE
Openstack terraform changes: Add (optional) dns_nameservers, add volu…

### PR DESCRIPTION
**Pull request by paulfrybarger-tgt**
…me size parameters, and allow ssh_user passthrough to modules.

### CODE REVIEW

**Comment by siddharthist** [comment]
Does this require users to manually set the `ssh-user`? I think we should have some kind of default.

**Comment by siddharthist** [comment]
This looks like a good cleanup! Thanks @paulfrybarger-tgt.

File: terraform/openstack-modules.sample.tf [hunk 1]
```diff
@@ -14,6 +15,9 @@ variable floating_ip_pool { default = "name-of-your-floating-ip-pool" }
 # Run 'nova image-list' to get your image name
 variable image_name  { default = "your-CentOS-7" }
 
+# DNS servers passed to Openstack subnet
+variable dns_nameservers { default = "" } # comma separated list of ips, e.g. "8.8.8.8,8.8.4.4"
```
Comment (stevendborrelli) [inline terraform/openstack-modules.sample.tf]: If this is not set, does openstack use the default values?

Comment (pfrybar) [inline terraform/openstack-modules.sample.tf]: Yeah, I use a `compact(split(...))` to turn the empty string into an empty array, which will treat dns_nameservers as if it was not set at all.

**Comment by stevendborrelli** [comment]
Thanks!

**Pull request merged**